client/my-sites/hosting/main.js

```diff
@@ -143,11 +142,6 @@ const MainCards = ( {
 			content: <CacheCard disabled={ isBasicHostingDisabled } />,
 			type: 'basic',
 		},
-		siteId && {
-			feature: 'wp-admin',
-			content: <SiteAdminInterfaceCard siteId={ siteId } />,
```

Comment:
I think for now we should still keep this admin interface style setting duplicated in Hosting. We have been sending early access users instructions on how to enable the classic view in Hosting. Removing this here will only lead to confusion.

After we have rolled out to all Atomic classic then we can remove the duplication. What do you think? cc: @lsl @autumnfjeld

Reply:
Sounds good. I agree we can remove the duplication later.

Reply:
Kept the duplication by ebf2706a1f25d1e15d20f7c3de42672d8f73c4c4

Reply:
Yep, I duplicated it by https://github.com/Automattic/wp-calypso/pull/88459/commits/ebf2706a1f25d1e15d20f7c3de42672d8f73c4c4 🙂